### PR TITLE
[14.0] [mig] document_quick_access_folder_auto_classification

### PR DESCRIPTION
**Pull request by schhatbar-initos**

### CODE REVIEW

File: document_quick_access_folder_auto_classification/security/ir.model.access.csv [hunk 1]
```diff
@@ -1,2 +1,3 @@
 id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
 access_document_quick_access_missing,access_document_quick_access_missing,model_document_quick_access_missing,group_missing_document,1,1,1,0
+access_document_quick_access_missing_assign,access_document_quick_access_missing_assign,model_document_quick_access_missing_assign,base.group_user,1,0,0,0
```
Comment (etobella) [inline document_quick_access_folder_auto_classification/security/ir.model.access.csv]: With this permissions, a standard user might not be able to do it. Can you check it please?

**Review by etobella — CHANGES_REQUESTED** [review]

**Review by dsolanki-initos — APPROVED** [review]
Functional test and code review. LGTM.

**Comment by schhatbar-initos** [comment]
@etobella Can you review this PR?
Thanks in Advance!

**Comment by etobella** [comment]
Hi @schhatbar-initos I was thinking that an upgrade can be done in order to simplify all the process. This way, glue modules like https://github.com/OCA/server-ux/pull/300 will be managed by storage and it could

I will propose a refactoring on 13 soon (today or tomorrow if it goes as expected), but It will need storage and edi_storage. WDYT?

**Comment by schhatbar-initos** [comment]
Hi @etobella 
Why do we need to do refactoring for v13?
I think storage and edi_storage is not used with this module.

**Comment by etobella** [comment]
I know it is not, but, this way, we can create a more harmonized method here. This is the main idea. This way, we don't need glue modules for other kind of implementations (SSH or files from other kinds of storages)

**Comment by fshah-initos** [comment]
Hi @etobella Please let us know once the refactoring in v13 is completed.

**Comment by etobella** [comment]
You can review it on #323

**Comment by fshah-initos** [comment]
Is it possible to merge this PR?

**Comment by fshah-initos** [comment]
It would be great if we could merge the PR.

File: document_quick_access_folder_auto_classification/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+{
+    "name": "Document Quick Access Folder Auto Classification",
+    "summary": """
+        Auto classification of Documents after reading a QR""",
+    "version": "14.0.1.0.0",
+    "license": "AGPL-3",
+    "author": "Creu Blanca,Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/server-ux",
+    "depends": ["document_quick_access", "edi_storage"],
```
Comment (etobella) [inline document_quick_access_folder_auto_classification/__manifest__.py]: now, it is edi_storage_oca, now that it has been installed, can you check it?

**Review by etobella — CHANGES_REQUESTED** [review]

**Comment by fshah-initos** [comment]
@etobella I've updated it can you please check now?

**Comment by etobella** [comment]
Travis is not happy, can you check it please?

**Comment by etobella** [comment]
Travis is complaining because a library is missing. Check them on travis config file from 13.0

https://github.com/OCA/server-ux/blob/13.0/.travis.yml#L15

**Comment by etobella** [comment]
I think he also needs zbar-tools

**Comment by hkapatel-initos** [comment]
> I think he also needs zbar-tools

can you please help me to resolve Travis issue?

**Comment by hkapatel-initos** [comment]
Hello @etobella Can you please re-review the PR.

File: document_quick_access_folder_auto_classification/models/ir_attachment.py [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+
+class IrAttachment(models.Model):
+    _inherit = "ir.attachment"
+
+    def _search_document_pdf(self):
+        self.ensure_one()
+        records = []
+        try:
+            images = pdf2image.convert_from_bytes(base64.b64(self.datas))
```
Comment (etobella) [inline document_quick_access_folder_auto_classification/models/ir_attachment.py]: base64.b64decode

**Comment by hkapatel-initos** [comment]
> _No description provided._

Previous version: https://github.com/OCA/server-ux/pull/323

**Comment by hkapatel-initos** [comment]
Hello @etobella it would be nice if you check this pull.

**Comment by etobella** [comment]
@hkapatel-initos I made a new PR amending your commits fixing it. You can either take my changes and apply them here or just close this one and approve mine :wink:

**Comment by hkapatel-initos** [comment]
> @hkapatel-initos I made a new PR amending your commits fixing it. You can either take my chances and apply them here or just close this one and approve mine wink

it would be great if we add changes here because in this pull you can see that already been approved by one reviewer and it would be great that after I applied this fixing you can give approval to this pull so we can merge this module ASPS.

**Comment by hkapatel-initos** [comment]
Hey, @etobella Thanks, 
To make Travis happy I add packages in travis.yml, changes are done,  As second reviewer  it would be great if you gave approval to this pull

File: document_quick_access_folder_auto_classification/__manifest__.py [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+    "depends": ["document_quick_access", "edi_storage_oca"],
+    "data": [
+        "security/security.xml",
+        "security/ir.model.access.csv",
+        "wizards/document_quick_access_missing_assign.xml",
+        "views/document_quick_access_missing.xml",
+        "data/config_parameter.xml",
+        "data/cron_data.xml",
+    ],
+    "maintainers": ["etobella"],
```
Comment (etobella) [inline document_quick_access_folder_auto_classification/__manifest__.py:22]: Add the external dependancy and we will be done

Comment (hkapatel-initos) [reply document_quick_access_folder_auto_classification/__manifest__.py:22]: Done

**Review by etobella — APPROVED** [review]

**Review by hkapatel-initos — APPROVED** [review]

**Comment by hkapatel-initos** [comment]
Hello, @LoisRForgeFlow Can we merge this?

**Pull request merged**